borgmatic/actions/bootstrap.py

```diff
@@ -0,0 +1,80 @@
+import logging
```

Comment:
Not a big deal at all right now, but you might consider moving this file to `borgmatic/actions/config/bootstrap.py`.